.github/workflow_scripts/lint_check.sh

```diff
@@ -4,6 +4,7 @@ cd ../../
 set -ex
 
 python3 -m pip install --upgrade prospector pip
+yes | pip3 install astroid==v3.0.0
```

Comment:
Is this change related to this PR? If not let's do rebase to ensure only changes related to the PR show up.

Reply:
With out this, there is a change the lint check will fail due to version miss-match between astroid and pylint.